tests/NLog.UnitTests/LayoutRenderers/Contexts/AllEventPropertiesTests.cs

```diff
@@ -166,6 +167,39 @@ public void AllEventWithFluent_without_callerInformation()
             AssertDebugLastMessage("m", "Test=InfoWrite, coolness=200%, a=not b");
         }
 
+        [Fact]
+        public void WithPropertiesTest()
```

Comment:
Please refactor the code to use `LogFactory.Setup.LoadConfiguration(builder => )`. You can search for other unit-tests for guidance of how to use it (You are very wellcome to also fix the old code that you copied)